.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think your IDE created this file accidentally by the way

Reply:
Thank you, that is fixed!